### PR TITLE
Add new package prbt_gazebo

### PR DESCRIPTION
**Pull request by agutenkunst**
as example how to use our model in combination with Gazebo

### CODE REVIEW

File: prbt_gazebo/CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package prbt_gazebo
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Forthcoming
+-----------
+* Created trajectory generation package with ptp, lin, circ and blend planner
```
Comment (martiniil) [inline prbt_gazebo/CHANGELOG.rst]: Please adapt changelog

File: prbt_gazebo/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# prbt_gazebo
+Package providing a launch which can be used to launch Gazebo with the
```
Comment (martiniil) [inline prbt_gazebo/README.md]: launch <i>file</i>?

File: prbt_gazebo/launch/sim_prbt_gazebo.launch [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<!--
+Copyright (c) 2018 Pilz GmbH & Co. KG
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
```
Comment (martiniil) [inline prbt_gazebo/launch/sim_prbt_gazebo.launch:5]: License file missing

**Review by martiniil — CHANGES_REQUESTED** [review]
A travis job failed. Can you have a look at it?

File: prbt_gazebo/launch/sim_prbt_gazebo.launch (hunk 3 above)
Comment (agutenkunst) [reply prbt_gazebo/launch/sim_prbt_gazebo.launch:5]: Fixed.

File: prbt_gazebo/README.md (hunk 2 above)
Comment (agutenkunst) [reply prbt_gazebo/README.md]: Fixed.

File: prbt_gazebo/CHANGELOG.rst (hunk 1 above)
Comment (agutenkunst) [reply prbt_gazebo/CHANGELOG.rst]: Fixed.

Comment (agutenkunst) [reply prbt_gazebo/CHANGELOG.rst]: Adapted.

**Review by martiniil — APPROVED** [review]

**Pull request merged**